bridge/config/config.go

```diff
@@ -166,6 +166,7 @@ type Protocol struct {
 	URL                    string     // mattermost, slack // DEPRECATED
 	UseAPI                 bool       // mattermost, slack
 	UseLocalAvatar         []string   // discord
+	UseNativeUpload        bool       // discord
```

Comment:
I think this behaviour should be non-configurable (i.e. just the default behaviour) for any bridges where it's possible. I can't think of a good use-cases for some people wanting external links, and others _not_ wanting external links.

Reply:
(We can always add a setting later on if someone really wants it. Otherwise I don't think it's worth supporting an extra code path here.)

Reply:
The reason why I had it configurable is because according to the documentation in the wiki a public link is preferable (referenced in the my issue #1965). 

For most platforms this actually isn't possible (without using the media server)
Discord is unique in this regard.

So we'd be betraying that "standard" by making it non configurable.

Personally, I'm fine with making it the default discord behavior but it's up to the maintainers/community if that's the route we take.

Reply:
my bad, i missed that part of the documentation. i will post a comment on your issue to help inform the spec for this pull request

Reply:
So I updated the CR to not make it an option, should drop the commit for that change?

Reply:
> So I updated the CR to not make it an option, should drop the commit for that change?

@42wim what do you think?